diff --git a/node_modules/hyperid/hyperid.js b/node_modules/hyperid/hyperid.js
index 7f32004..41fe5f6 100644
--- a/node_modules/hyperid/hyperid.js
+++ b/node_modules/hyperid/hyperid.js
@@ -3,7 +3,8 @@
 const uuidv4 = require('./uuid-node')
 const parser = require('uuid-parse')
 const maxInt = Math.pow(2, 31) - 1
-const Buffer = require('buffer').Buffer
+const process = require('process');
+const Buffer = process.env && process.env.BUN_TEST ? require('buffer').Buffer : require('buffer').default;
 const base64Padding = Buffer.from('==', 'base64')
 
 function hyperid (opts) {
